### PR TITLE
dxScheduler - add argument 'e.popup' to onAppointmentFormOpening

### PR DESCRIPTION
**Pull request by andrewmakarov**

### CODE REVIEW

File: js/ui/scheduler.d.ts [hunk 1]
```diff
@@ -394,7 +395,7 @@ export interface dxSchedulerOptions extends WidgetOptions<dxScheduler> {
      * @prevFileNamespace DevExpress.ui
      * @public
      */
-    onAppointmentFormOpening?: ((e: { component?: dxScheduler, element?: dxElement, model?: any, appointmentData?: any, form?: dxForm, cancel?: boolean }) => any);
+    onAppointmentFormOpening?: ((e: { component?: dxScheduler, element?: dxElement, model?: any, appointmentData?: any, form: dxForm, popup: dxPopup, cancel: boolean }) => any);
```
Comment (RomanTsukanov) [inline js/ui/scheduler.d.ts]: The doccomment is missing the `popup` parameter.

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

**Review by RomanTsukanov — DISMISSED** [review]
Doccomment is fine

**Review by AryamnovEugeniy — APPROVED** [review]

**Review by AntonSermyazhko — APPROVED** [review]

**Review by Yarovaya — APPROVED** [review]

**Review by IlyaKhD — CHANGES_REQUESTED** [review]
`update-ts` failed to update dx.all.d.ts properly, I will fix it in a few days

**Comment by IlyaKhD** [comment]
> `update-ts` failed to update dx.all.d.ts properly, I will fix it in a few days

Actually we can merge this PR and create another one once tools are ready

**Review by IlyaKhD — APPROVED** [review]

**Pull request merged**